<html>
<head><title>LigPlot+ Operating Manual</title></head>
<body bgcolor="white">
<center>

<h1>LigPlot<SUP>+</SUP> v.1.3</h1>
<h1>Graphical User Interface for the LIGPLOT and DIMPLOT programs</h1>
<h2>Operating Manual</h2>
<P>
<i><u>Contents</u></i>
<table bgcolor=white cellspacing=0 cellpadding=0>
 <tr>
  <td rowspan=11 align=center valign=bottom>
   <img src="ligplot_3tmn.gif"><br>
   <font size=-1>
   LIGPLOT of Val-Trp dipeptide<br> in 3tmn</font>
  </td>
  <td rowspan=11><img height=1 width=20 src="1x1.gif"></td>
  <td align=right valign=top>1.</td>
  <td>&nbsp;</td>
  <td valign=top>Introduction</td>
  <td rowspan=11><img height=1 width=20 src="1x1.gif"></td>
  <td rowspan=11 align=center valign=bottom>
   <img src="dimplot_3sdp.gif"><br>
   <font size=-1>
   DIMPLOT of interface between<br> chains A and B in 3sdp</font>
  </td>
 </tr>
 <tr>
  <td align=right valign=top>2.</td>
  <td>&nbsp;</td>
  <td valign=top>Paths and directories</td>
 </tr>
 <tr>
  <td align=right valign=top>3.</td>
  <td>&nbsp;</td>
  <td valign=top>Generating a LIGPLOT/DIMPLOT diagram</td>
 </tr>
 <tr>
  <td align=right valign=top>4.</td>
  <td>&nbsp;</td>
  <td valign=top>Editing the plot</td>
 </tr>
 <tr>
  <td align=right valign=top>5.</td>
  <td>&nbsp;</td>
  <td valign=top>Multiple plots</td>
 </tr>
 <tr>
  <td align=right valign=top>6.</td>
  <td>&nbsp;</td>
  <td valign=top>Plot parameters</td>
 </tr>
 <tr>
  <td align=right valign=top>7.</td>
  <td>&nbsp;</td>
  <td valign=top>Adding missing H-bonds</td>
 </tr>
 <tr>
  <td align=right valign=top>8.</td>
  <td>&nbsp;</td>
  <td valign=top>Runtime parameters</td>
 </tr>
 <tr>
  <td align=right valign=top>9.</td>
  <td>&nbsp;</td>
  <td valign=top>Removing unwanted H-bonds</td>
 </tr>
 <tr>
  <td align=right valign=top>10.</td>
  <td>&nbsp;</td>
  <td valign=top>Plots from structural alignments</td>
 </tr>
 <tr>
  <td align=right valign=top>11.</td>
  <td>&nbsp;</td>
  <td valign=top>Structural alignment file formats</td>
 </tr>
</table>
</center>
<p>

<h3>1. Introduction</h3>

LigPlot<SUP>+</SUP> is a graphical front-end to the LIGPLOT and
DIMPLOT programs. The plots generated by these programs can
be interactively edited on screen, superposed and printed.
<ul>
<li><B>LIGPLOT</B> - automatically generates schematic diagrams of
protein-ligand interactions for a given ligand in a PDB file.

<li><B>DIMPLOT</B> - plots interactions across a selected protein-protein or
domain-domain interface.
</ul>

The very first time you run LigPlot<SUP>+</SUP> on your computer, you
will need to define several paths and directories so that the program
knows where to find things like PDB files, the Het Group Dictionary
and the RasMol program (if you have it). LigPlot<SUP>+</SUP> will
detect that it is being run for the first time and will pop up an
entry-form, described in the section below, for you to fill in.

<h3>2. Paths and directories</h3>

The entry-form for defining paths and directories looks as shown below.

<table>
 <tr>
  <td>
   <img src="paths_form.gif">
  </td>
 </tr>
</table>

<P>
Edit the text fields as follows:

<P>
<B>PDB paths.</B><BR>

PDB files are most conveniently identified simply by the 4-character
PDB code or identifier. If your system contains one or more locations
where PDB files are stored, you can define a "template" path which
will allow <B>LIGPLOT<SUP>+</SUP></B> to find the relevant PDB file
from its 4-character code alone. The files can be gzipped or not.

<P>
Alternatively, if you are connected to the internet when you run
<B>LIGPLOT<SUP>+</SUP></B>, the required file can be retrieved by ftp
from a given ftp site.

<P>
The 4 characters of the PDB code in each template are represented as
<font color=green><B>abcd</B></font>. These appear in the template
within square brackets.

<P>
For example, the template

<BLOCKQUOTE>
<font color=green><B>C:/roman/pdbsum/pdb/pdb[abcd].ent</B></font>
</BLOCKQUOTE>

means that, for example, PDB code <B>1ral</B> would be translated as:

<BLOCKQUOTE>

<B>C:/roman/pdbsum/pdb/pdb1ral.ent</B>
</BLOCKQUOTE>

Similarly, the ftp template

<BLOCKQUOTE>
<font color=green><B>ftp://ftp.ebi.ac.uk/pub/databases/rcsb/pdb-remediated/data/structures/divided/pdb/[bc]/pdb[abcd].ent.gz</B></font>
</BLOCKQUOTE>

would map to PDB code <B>1ral</B> as:

<BLOCKQUOTE>
<B>ftp://ftp.ebi.ac.uk/pub/databases/rcsb/pdb-remediated/data/structures/divided/pdb/ra/pdb1ral.ent.gz</B>
</BLOCKQUOTE>

Note that here the PDB files are stored in subdirectories whose names
correspond to the middle part of the PDB code (here shown as
"<B>[bc]</B>").

<P>
The above ftp address retrieves gzipped PDB files from the PDBe ftp
site at the EBI. A full list of possible ftp addresses is given below:

<BLOCKQUOTE>
<table border=0 cellpadding=0 cellspacing=0>
 <tr>
  <td>
   <B>PDBe</B>
  </td>
  <td>
   &nbsp; &nbsp;

  </td>
  <td>
   <font color=green><B>ftp://ftp.ebi.ac.uk/pub/databases/rcsb/pdb-remediated/data/structures/divided/pdb/[bc]/pdb[abcd].ent.gz</B></font>
  </td>
 </tr>
 <tr>
  <td>
   <B>PDBj</B>

  </td>
  <td>
   &nbsp; &nbsp;
  </td>
  <td>
   <font color=green><B>ftp://pdb.protein.osaka-u.ac.jp/v3/pub/pdb/data/structures/divided/pdb/[bc]/pdb[abcd].ent.gz</B></font>
  </td>
 </tr>

 <tr>
  <td valign=top>
   <B>RCSB&nbsp;PDB</B>
  </td>
  <td>
   &nbsp; &nbsp;
  </td>
  <td valign=top>

   <font color=green><B>ftp://ftp.wwpdb.org/pub/pdb/data/structures/divided/pdb/[bc]/pdb[abcd].ent.gz</B></font>
  </td>
 </tr>
</table>
</BLOCKQUOTE>

<P>
<B>The Het Group Dictionary.</B><BR>

<B>LIGPLOT<SUP>+</SUP></B> gets its information about ligand molecules
from the Het Group Dictionary. This defines the atom names,
connectivities and bond orders of every Het Group in the PDB. You can
download a copy of this dictionary from the <B>wwPDB</B> at:


<BLOCKQUOTE>
<a href="ftp://ftp.wwpdb.org/pub/pdb/data/monomers/components.cif"><B>ftp://ftp.wwpdb.org/pub/pdb/data/monomers/components.cif</B></a>
</BLOCKQUOTE>

<P>
It is a good idea to download it regularly to always have an
up-to-date copy.

<P>
Record the full path and filename of the Het Group Dictionary here so
that the programs know where to find it.

<P>
<B>SDF Dictionary.</B> Alternatively, if your ligand definitions are
in a <B>.sdf</B> file, you can provide the name of your file
here. There are two important considerations:

<OL>
<LI> The file must have a <B>.sdf</B> extension.
<LI> The 3-character "residue name" of the ligand, as it appears in
your PDB file(s), must be given in the molecule's description. For
example:<BR>

<font color=green><B>&gt; &lt;PDB_HET_ID&gt;<BR>
PLP
</B></font>
</OL>

The exact name of the field (in this case "PDB_HET_ID") is not
important as different people will use different conventions. The
program will look for potential identifiers and, if they match one of
the Het Group names in the PDB file will take the corresponding
molecule definition.

<P>
<B>Temporary directory.</B><BR>

<B>LIGPLOT<SUP>+</SUP></B> needs a directory where it can puts its
temporary working files (which it deletes once it has done with
them). Use this parameter to define a directory to which you have
write-access and which <B>LIGPLOT<SUP>+</SUP></B> can use for its
working files.

<P>
<B>Name of RasMol executable.</B><BR>

Enter the full path and name of the RasMol executable file. This will
allow you to display the 3D coords of the given <B>LIGPLOT/DIMPLOT</B>
diagrams. If you do not have RasMol, enter <font
color=green><B>NONE</B></font> here, although it is recommended that
you download and install a copy of the program.

<P>
Linux users might find it more convenient to enter the RasMol path using
the following format:

<BLOCKQUOTE>
<font color=green><B>xterm -e rasmol -script</B></font>
</BLOCKQUOTE>

This will open up an X-term window as well as a RasMol window. The former
can be used for entering RasMol commands.

<P>
<B>Save.</B> Press the <B>Save</B> button to store your paths. If you
should need to alter the paths at some leter date, you can do so via
the <B>Program paths</B> menu item in the <B>Edit</B> menu.

<table>
 <tr>
  <td>
   <img src="paths_select.gif">
  </td>
 </tr>
</table>

<h3>3. Generating a LIGPLOT/DIMPLOT diagram</h3>

To generate a new LIGPLOT or DIMPLOT diagram, select the PDB file of interest
using <font color=blue><B>File-Open-PDB file</B></font> from the menu bar.

<table>
 <tr>
  <td>
   <img src="open_pdb.gif">
  </td>
  <td>&nbsp;</td>
  <td>
   <img src="enter_pdbcode.gif">
  </td>
 </tr>
</table>

Enter the 4-character PDB code into the dialogue window that will pop
up. The program will for the PDB file using the location templates
defined in your <font color=purple><B>ligplus.par</B></font> file (see
Installation Instructions).

<P>
Alternatively, you can browse for a PDB file in your local directory
system by clicking the <font color=blue><B>Browse</B></font> button.

<P>
When the file has been loaded, you will see a summary of the ligands
and metals in the PDB file, if any, as shown below for PDB entry 1a95.

<table>
 <tr>
  <td>
   <img src="ligplot_tab.gif">
  </td>
  <td>&nbsp;</td>
  <td>
   <img src="dimplot_tab.gif">
  </td>
 </tr>
</table>

<P>
Clicking on the LIGPLOT or DIMPLOT tabs allows you to choose which of
the two programs to run.

<ul>
<li><B>LIGPLOT</B>
 <ol>
  <li>Select which ligand you wish to plot from the list.

<P>
Alternatively, you can enter your own residue range in the box. The
residue range can be one of the following:

<table>
 <tr>
  <td>
   <U><I>Example</I></U>
  </td>
  <td>&nbsp;</td>
  <td>
   <U><I>Description</I></U>
  </td>
 </tr>
 <tr>
  <td valign=top>
   <B>18&nbsp;20&nbsp;A</B>
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   Residue range: start-residue-no. end-residue-no. chain-id<BR>
   If the chain-id is blank it should be omitted
  </td>
 </tr>
 <tr>
  <td valign=top>
   <B>NAG&nbsp;18&nbsp;MAN&nbsp;20&nbsp;A</B>
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   Residue names and numbers, plus chain id.
  </td>
 </tr>
 <tr>
  <td valign=top>
   <B>-n818&nbsp;108&nbsp;-n818&nbsp;108&nbsp;A</B>
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   Where the residue name is numeric, prefix it with "-n" to indicate
   that it is the residue name rather than the residue number.
  </td>
 </tr>
 <tr>
  <td valign=top>
   <B>"MTE"&nbsp;1&nbsp;"&nbsp;&nbsp;G"&nbsp;2&nbsp;B</B>
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   Where a residue name includes leading blanks (eg as in the old
   definition of the nucleic acid bases, A, C, G and T), enclose it in
   quotes and include the spaces.
  </td>
 </tr>
 <tr>
  <td valign=top>
   <B>"-nMTE"&nbsp;1&nbsp;"-n&nbsp;&nbsp;G"&nbsp;2&nbsp;B</B>
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   An alternative for the case above.
  </td>
 </tr>
</table>

  <li>To include waters on the plot, tick the checkbox next to the
      "<B>Include waters</B>" option.

<P>
  <li>Tick the "Filter waters" checkbox to remove waters that don't make
      at least 2 H-bonds to ligand and/or protein (eg this removes
      waters that interact only with another water, or with just the protein).

  </ol>

<P>

<li><B>DIMPLOT</B>
 <ol>
  <li>Select which protein chain pair you would like
plotted, or enter the residue ranges of the chains or domains you want
to plot. Enter each residue range as: <B>N1-N2&nbsp;C</B>, where <B>N1</B>
is the start residue, <B>N2</B> is the end residue, and <B>C</B> is
the chain id. If a domain is split, you can define it using several
residue ranges, separating each range with an ampersand. For example

<BLOCKQUOTE>
<B>1-136 A & 338-371 A</B>
</BLOCKQUOTE>

  <li>To include waters on the plot, tick the checkbox next to the
      "<B>Include waters</B>" option.

  </ol>
</ul>

<P>
Then to run either program, click on the <font color=blue><B>Run</B></font>
button.

<P>
When the plot has been generated it will appear on screen and can now
be edited interactively, as described next.

<h3>4. Editing the plot</h3>

<h3>a. Moving residues, atoms, and text items</h3>

To move any of the objects on the plot click and drag the object using
the left mouse button.
<p>
Note, that to switch between moving whole residues and single atoms click
the <font color=blue><B>Move residue/move atom</B></font> button at
the bottom-right of the frame to toggle between the two modes.
<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="moveres_button.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=bottom>
   <img src="moveat_button.gif">
  </td>
 </tr>
</table>

<p>
Note also that the hydrophobic contact groups cannot be moved in the 
<font color=blue><B>Move atom</B></font> mode.

<h3>b. Panning and zooming</h3>

Use the left mouse button to click-and-drag on any blank area of the
plot to pan around the plot.
<P>
Use the right mouse button to click-and-drag on any blank area of the
plot to zoom in and out of the plot.

<h3>c. Recentre</h3>

You can recentre the plot at any time by clicking on the <font
color=blue><B>Recentre</B></font> button at the bottom left of the
LigPlot<SUP>+</SUP> window.

<table>
 <tr>
  <td valign=bottom>
   <img src="recentre_button.gif">
  </td>
 </tr>
</table>

<h3>d. Rotating residues</h3>

You can rotate a residue about any of its atoms by right-clicking on
the atom you want to pivot about. A marker will appear
over the selected atom to identify it.

<table>
 <tr>
  <td valign=bottom>
   <img src="rotate_res1.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=bottom>
   <img src="rotate_res2.gif">
  </td>
 </tr>
</table>

<p>
To rotate about this atom, click-and-drag any other atom in the
residue. Release the button when you have reached the desired
position.

<P>
To deselect the atom, right- or left-click on any blank part of the
plot, or on any other residue.

<h3>e. Flipping about a bond</h3>

You can flip a residue, or part or a residue, about any of its bonds.
<P>
Firstly, select the bond by clicking on it with the right
mouse button. A marker will appear over the selected bond.

<table>
 <tr>
  <td valign=bottom>
   <img src="flip_res1.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=bottom>
   <img src="flip_res2.gif">
  </td>
 </tr>
</table>

<p>
To flip about this bond, click with the left mouse-button on any
atom on the side of the residue to be flipped. A single click will
perform the flip. Clicking a second time on any of the flipped atoms
will reverse the flip.
<P>
To "unselect" the flip-bond, click with the right or middle mouse
button on any blank area of the plot. The marker will disappear.

<h3>f. Editing text labels</h3>

You can edit any of the text labels on the plot.
<P>

Select the text item by right-clicking on it. A box containing the
text string will appear. You can edit the text as required. Click the
<font color=blue><B>OK</B></font> button to accept the new version, or
the <font color=blue><B>Cancel</B></font> button to retain the
previous text label.

<h3>5. Multiple plots</h3>

You can edit several related, and overlaid, plots at the same, moving
their components independently and switching between them. The
currently active plot is shown in full colour in the foreground, while
all background plots are greyed out. Before printing you can "switch
off" the display of the background plots such that only the foreground
plot appears. This allows you to produce a set of clean plots with
equivalent components in equivalent positions on them. Alternatively,
if you use the "Write PostScript" option, you can have each plot
printed on a separate page.

<P>
The first plot is generated in the usual way. Each subsequent plot is
fitted to the first by a sequence-based comparison of the two binding
sites. This identifies equivalent residues in the 3D structures and
these equivalences drive the generation of the second LIGPLOT. If the
program is unable to match the binding sites based on their sequences,
it tries to fit the ligands using a graph matching procedure. For very
distantly related proteins it is better to use a structural
alignment. Section 7 describes how to import such an alignment.

<h3>a. Loading additional plots</h3>

In the simplest case, additional plots are loaded/generated in the
same way as the initial plot was. Use
either <font color=blue><B>File-Add</B></font>
or <font color=blue><B>File-Open-PDB file</B></font> to fit a new
LIGPLOT on top of the current one on screen.

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="add_replace.gif">
  </td>
 </tr>
</table>

<P>
The new plot, when generated, will become the foreground plot; any
current plot(s) will go into the background.

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="oneplot.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=bottom>
   <img src="two_plots.gif">
  </td>
 </tr>
</table>

<p>
The left-hand example above shows a plot of the interactions between
guanine 600 in PDB entry 2pwu and the protein residues. The right-hand
plot shows superposed on it a plot of a similar molecule (9DG) bound to
PDB entry 1q2r. The red circles and ellipses identify the residue on
the latter plot that are equivalent to the underlying residues from
the first plot.

<p>
LigPlot<SUP>+</SUP> attempts, as best it can, to place
residues in the new plot on top of the equivalent residues in the old
one. It does this by first performing a simple sequence alignment
between the set of interacting residues from each structure. The
equivalenced residues in this alignment are then used to superpose the
3D coordinates from both PDB files. This may throw up more pairs of
equivalenced residues, being those that significantly overlap in the
superposition. The residue equivalences are then used to "drive" the
generation of the new LIGPLOT; the 2D locations
from the first plot restrain the positions of the corresponding
residues in the second plot.

<P>
The superposition may not always be successful if either the two
proteins are quite different, or the second ligand is very large
relative to the first.

<P>
The "Split screen" button at the bottom of the frame separates the
two (or more) plots, and arranges them in the window.

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="splitscreen_button.gif">
  </td>
 </tr>
</table>

<P>
Two plots, as in the above example, are shown side by side:

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="split_plot.gif">
  </td>
 </tr>
</table>

<P>
The "Merge back" button will restore the plots to overlap mode.

<P>
You can change how the equivalent residues are shown on the plot by
using the <B>On/Off</B> parameters to switch off the red circles and,
instead, ticking the "<B>Highlight equivalent side chains</B>"
option. The resultant plot will look like:

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="split_plot_nocirc.gif">
  </td>
 </tr>
</table>

<P>
Here, the equivalent residues have a red underlay beneath their bonds
and atoms. Equivalent residues engaged in hydrophobic interactions are
shown in thicker lines.

<h3>b. Moving any of the split plots relative to others</h3>

When the plots are in split-screen mode, as above, you can move one
plot relative to the other(s) by selecting the "Move plot" option from
the button at the bottom right of the screen:

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="moveplot_button.gif">
  </td>
 </tr>
</table>

<P>
You can then move any of the plots on screen by clicking and dragging
any of their components (ie atoms, bonds, text items, etc). Remember
to revert to "Move residue" mode when you're done to return to normal
editing. When you merge the plots back, the "Move plot" option is lost.


<h3>c. Selecting the active plot</h3>

When the plots are merged, you can bring any of the background plots
into the foreground either by clicking on their greyed-out label in
the top right-hand corner, or by clicking on any of their components
in the plot itself (ie atoms, bonds, text items, etc).

<h3>d. Viewing structures in RasMol or PyMOL</h3>

If you have either <B>RasMol</B> or <B>PyMOL</B> installed, and have
defined the path to the program(s) in the <B>Paths and Directories</B>
above (Section 2), then there will be a <B>RasMol</B> and/or
<B>PyMOL</B> button at the bottom of the frame.

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="rasmol_button.gif">
  </td>
  <td><img height=1 width=20 src="1x1.gif"></td>
  <td valign=bottom>
   <img src="pymol_button.gif">
  </td>
 </tr>
</table>

<P>
Clicking on one of these buttons will pop up a <B>RasMol</B> or
<B>PyMOL</B> window containing the superposed ligands, in 3D, with
H-bonds added in cyan and atoms involved in non-bonded contacts
represented by dot surfaces (in RasMol) or transparent surfaces (in
PyMOL).

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="rasmol_2pwu_1q2r.gif">
  </td>
  <td><img height=1 width=20 src="1x1.gif"></td>
  <td valign=bottom>
   <img src="pymol_2pwu_1q2r.gif">
  </td>
 </tr>
</table>

<h3>e. Switching off the background plots</h3>

For printing, you might wish to switch off the background plots. You
can do this either by choosing <font color=blue><B>Edit-On/off
selection</B></font> from the menu bar, or by clicking on the <font
color=blue><B>On/off</B></font> button at the bottom of the frame.

<P>
<table>
 <tr>
  <td valign=top>
   <img src="onoff_select.gif">
  </td>
  <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
  <td valign=top>
   <img src="onoff_button.gif">
  </td>
 </tr>
</table>

<p>
Then, deselect the <font color=blue><B>Show inactive plots</B></font>
option, circled in red below.

<P>
<table>
 <tr>
  <td valign=bottom>
   <img src="onoff_dialogue_a.gif">
  </td>
 </tr>
</table>

<h3>6. Plot parameters</h3>

You can adjust a number of parameters that define the appearance of
the plot. These can be accessed either by using the <font
color=blue><B>Edit</B></font> option in the menu bar at the top of
the LigPlot<SUP>+</SUP> window, or clicking one of the buttons at the 
bottom left. These are shown in the table below.

<P>
<table>
 <tr>
  <td valign=top>
   <img src="onoff_select.gif">
  </td>
  <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
  <td valign=top>
   <img src="sizes_select.gif">
  </td>
  <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
  <td valign=top>
   <img src="colours_select.gif">
  </td>
 </tr>
 <tr>
  <td colspan=6><img height=20 width=1 src="1x1.gif"></td>
 </tr>
 <tr>
  <td valign=top>
   <img src="onoff_button.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   <img src="sizes_button.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   <img src="colours_button.gif">
  </td>
  <td>&nbsp;</td>
 </tr>
</table>

<P>
The colour options differ slightly depending on whether your plot is
a <B>LIGPLOT</B> or a <B>DIMPLOT</B>. For the former, bonds and atoms
are classified as <B>ligand</B> or <B>non-ligand</B>. For the latter,
the two classes are: <B>interface 1</B> and <B>interface 2</B>.

<P>
If you have two or more plots overlaid, you will be asked, after
making your changes, whether the new parameters are to apply to the
currently selected plot only, or to all plots. Thus you can, say, have
the bonds or labels shown in a different colour in each plot. The
background parameters always apply to all plots.

<h3>a. On/off selection</h3>

<table>
 <tr>
  <td valign=bottom>
   <img src="onoff_dialogue.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   <img src="onoff_dialogue_dim.gif">
  </td>
 </tr>
</table>

The on/off selection dialogue allows you to switch various components
of the plot on or off. If the box is checked, that item is switched
on. The left-hand panel above shows the <B>LIGPLOT</B> options while
the right-hand panel contains the <B>DIMPLOT</B> options.

<P>
The items are grouped into:
<font color=blue><B>Labels</B></font>,
<font color=blue><B>Atoms</B></font>,
<font color=blue><B>Bonds</B></font>
and
<font color=blue><B>Miscellaneous</B></font>.

<h3>b. Sizes selection</h3>

<table>
 <tr>
  <td valign=bottom>
   <img src="sizes_dialogue.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   <img src="sizes_dialogue_dim.gif">
  </td>
 </tr>
</table>

This dialogue allows you to alter the sizes of various objects on the
plot. The sizes are given in "&Aring;ngstr&ouml;ms" so as to relate
to the scale of the coordinates and bond lengths in the original
molecules.

<P>
The items are grouped into:
<font color=blue><B>Label sizes</B></font>,
<font color=blue><B>Atom and residue sizes</B></font>
and
<font color=blue><B>Bond widths</B></font>.

<P>
The left-hand panel above shows the <B>LIGPLOT</B> options while
the right-hand panel contains the <B>DIMPLOT</B> options.

<h3>c. Colour selection</h3>

<table>
 <tr>
  <td valign=top>
   <img src="colours_dialogue2.gif">
  </td>
  <td>&nbsp;</td>
  <td valign=top>
   <img src="colours_dialogue_dim.gif">
  </td>
 </tr>
</table>

This dialogue controls the colours of the objects on the plot. The
tabs give the colour selections for different item types:

<P>
<font color=blue><B>Background</B></font>,
<font color=blue><B>Labels</B></font>,
<font color=blue><B>Atoms</B></font>
and
<font color=blue><B>Bonds</B></font>.

<P>
The left-hand panel above shows the <B>LIGPLOT</B> options while
the right-hand panel contains the <B>DIMPLOT</B> options.

<P>
Note that, for ligand and non-ligand bonds there is a special extra
colour labelled "ATOM". If this "colour" is selected, the bonds will
be coloured such that each half is of the colour of the atom bonded at
that end.

<h3>7. Runtime parameters</h3>

The runtime parameters allow you to alter some of the parameters that
are used in generating the LigPlot<SUP>+</SUP> diagrams. The
parameters are accessed from the <font color=blue><B>Edit</B></font>
option in the menu bar at the top of the LigPlot<SUP>+</SUP> window.

<P>
<table>
 <tr>
  <td valign=top>
   <img src="runtime_option.gif">
  </td>
  <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
  <td valign=top>
   <img src="runtime_params.gif">
  </td>
 </tr>
</table>

<P>
The first two parameters relate to the <B>HBPLUS</B> program that
LigPlot<SUP>+</SUP> uses for computing all the potential hydrogen
bonds and non-bonded contacts. These are the maximum hydrogen-acceptor
and donor-acceptor distances for defining what is a hydrogen bonds. By
increasing these values you can "bring in" additional interactions
which may be just outside the program's criteria for an H-bond.

<P>
The next parameter relates to the treatment of any CONECT records that
may be in your PDB file. The CONECT records define which atoms are
covalently bonded to one another. They are generally used for defining
the connectivity of the ligand. Sometimes, these records are incorrect
and give unfeasibly long bonds.

<P>
You can choose how the program should treat the CONECT records. The
default is to use them if they look sensible - that is, if they don't
give ridiculously long bond lengths. The second option is to ignore
these records altogether; the program will compute the ligand's
covalent bonds itself, using set distance cut-offs. The third option
is to accept all CONECT records, irrespective of the bond lengths
they give.

<h3>8. Adding missing H-bonds</h3>

The H-bonds and non-bonded contacts shown by <B>LIGPLOT</B> are
calculated by <b>HBPLUS</b>. Occasionally,
<b>HBPLUS</b> can miss an H-bond, particularly when it encounters a
ligand it does not recognize.

<P>
If you need your plot to have an H-bond that <b>HBPLUS</b> has missed, you can
add it as follows:

<OL>
<LI>Edit your PDB file and add the missing H-bonds, or non-bonded contacts,
as "<B>HHB</B>" and "<B>NNB</B>" records, respectively. These records
should come <i>before</i> the <B>ATOM</B> records in the file.

<P>
The format of each "<B>HHB</B>" or "<B>NNB</B>" record should be
formatted <i>exactly</i> as shown below, where the dots represent
blank spaces.

<PRE>
<B>Key   &lt;----Atom 1 --->     &lt;----Atom 2 --->    Dist</B>
<font color=blue><B>HHB...RRR.C.NNNNI.AAAA.....RRR.C.NNNNI.AAAA....DDDD</B></font>
</PRE>

<table>
 <tr>
  <td valign=center>
   where 
  </td>
  <td>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>HHB</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   defines the record type: <B>HHB</B> for H-bonds, <B>NNB</B> for
   non-bonded contacts
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>RRR</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is the 3-character residue name
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>C</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is chain identifier (which may be blank)
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>NNNN</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is the right-justified residue number (1-9999)
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>I</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is the insertion code, or blank if none
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>AAAA</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is the 4-character atom name in standard PDB format
  </td>
 </tr>
 <tr>
  <td colspan=2>&nbsp;</td>
  <td valign=center><font size=-1 color=blue><B>DDDD</B></font></td>
  <td>&nbsp;</td>
  <td valign=center>
   is the distance between the two atoms (to 2 places of decimals)
  </td>
 </tr>
</table>

<P>
Some example lines are given below:

<PRE>
<B>HHB...RRR.C.NNNNI.AAAA.....RRR.C.NNNNI.AAAA....DDDD</B>
<font color=blue><B>HHB   HIS A  231   N       ASP A  226   OD2    2.77</B></font>
<font color=blue><B>HHB   HIS A  231   ND1     ASP A  226   OD1    2.76</B></font>
<font color=blue><B>HHB   HIS A  231   NE2     THR A 1317   O      2.80</B></font>
<font color=blue><B>HHB   THR A 1317   N       ALA A  113   O      2.72</B></font>
<font color=blue><B>HHB   ALA A  113   N       PHQ A  317   O2     3.31</B></font>
</PRE>

<LI>After editing any required H-bonds or non-bonded contacts into your PDB
file, rerun LigPlot<SUP>+</SUP> on this new PDB file.

</OL>

<h3>9. Removing unwanted H-bonds</h3>

To remove any unwanted H-bonds or non-bonded contacts from your plot,
follow the steps above, but annotating the unwanted bonds by
"<B>-HB</B>" and "<B>-NB</B>" records in your PDB file,
respectively. The value you put in the "<B>Dist</B>" is not important
and can be "<B>0.00</B>".

<P>
So, for example, to remove the H-bond between the
<b>NH<SUB>2</SUB></b> of
<b>Arg286(A)</b> and the <b>O7</b> of your ligand, <b>LIG1(B)</b>, you
would add the following line to your PDB file:


<BLOCKQUOTE>
<PRE>
<B>-HB...RRR.C.NNNNI.AAAA.....RRR.C.NNNNI.AAAA....DDDD</B>
<font color=blue><B>-HB   ARG A  286   NH2     LIG B    1   O7     0.00</B></font>
</PRE>
</BLOCKQUOTE>

<h3>10. Plots from structural alignments</h3>

For distantly related proteins it may be difficult for
LigPlot<SUP>+</SUP> to reliably identify equivalent residues in the
binding sites. Thus an overlaid plot of two or more structures might
not give a good alignment.

<P>
One way to help the program is to supply it with a structural
alignment between the proteins, letting it know which residues are
equivalent in 3D.

<P>
At present, LigPlot<SUP>+</SUP> only accepts structural alignments in
the following formats:

<ul>
<li>CORA format (ie from the CATH team's CORA program)

<li>CAF format (also from the CATH team's CORA program)

<li>Multiple FASTA format

</ul>

The formats are described later.

<P>
<b>Import the alignment</b>

<P>
Import using <font color=blue><B>File-Import</B></font> from the menu bar.

<table>
 <tr>
  <td valign=bottom>
   <img src="import.gif">
  </td>
 </tr>
</table>

Locate the appropriate <B>.cora</B>, <B>.caf</B> or <B>.fasta</B> file
and Open it.

<P>
A pop-up window will show you the list of PDB codes in the file and
allow you to select which one you want to plot first.

<table>
 <tr>
  <td valign=bottom>
   <img src="cora_list.gif">
  </td>
 </tr>
</table>

<P>
After the first plot has been generated, you can add others on top of
it via either <font color=blue><B>File-Add</B></font>
or <font color=blue><B>File-Open-PDB file</B></font>.

<h3>11. Structural alignment file formats</h3>

The following are the structural alignment file formats currently
accepted by LigPlot<SUP>+</SUP>:

<P>
<B>1. CORA format (.cora or .aln)</B>

<P>
The format must be CORA v.1.1, which is as follows:

<P>
<PRE>
#FM CORA_FORMAT 1.1
3
6insE0 1igl00 1bqt00
73
   1    0    1    0  0  0    1  A  0    0  0  0   0    0    0   0
   2    0    1    0  0  0    2  Y  0    0  0  0   0    0    0   0
   3    0    2    1B F  0    3  R  0    0  0  0   0    0    0   0
   4    0    3    2B V  H    4  P  0    1  G  0   0    1    0   2
   5    1    3    3B N  H    5  S  0    2  P  0   0    1    0   6
   6    0    3    4B Q  H    6  E  0    3  E  0   0    1    0   2
   7    2    3    5B H  H    7  T  0    4  T  0   0    1    0   5
---------+---------+---------+---------+---------+---------+---------+
1234567890123456789012345678901234567890123456789012345678901234567890
         1         2         3         4         5         6         7
</PRE>

<ul>
 <li>Header information:
  <ul>
   <li>Format and version: '#FM CORA_FORMAT 1.1'
   <li>Total number of proteins in the alignment
   <li>PDB codes identifying the proteins in the alignment with,
   optionally, chain identifier and domain number - eg <B>3d36A01</B>).
   <li>Total number of alignment positions
  </ul>
 <li>Alignment:

<PRE>
     START       PROT 1     PROT 2     PROT N         END
<------------><---------><---------><---------><---------------->
ddddxddddxddddxddddcxcxxcxddddcxcxxcxddddcxcxxcxxxcxddddxddddxxdd

   1    0    1    0  0  0    1  A  0    0  0  0   0    0    0   0
   2    0    1    0  0  0    2  Y  0    0  0  0   0    0    0   0
   3    0    2    1B F  0    3  R  0    0  0  0   0    0    0   0
   4    0    3    2B V  H    4  P  0    1  G  0   0    1    0   2
   5    1    3    3B N  H    5  S  0    2  P  0   0    1    0   6
   6    0    3    4B Q  H    6  E  0    3  E  0   0    1    0   2
   7    2    3    5B H  H    7  T  0    4  T  0   0    1    0   5
---------+---------+---------+---------+---------+---------+---------+
1234567890123456789012345678901234567890123456789012345678901234567890
         1         2         3         4         5         6         7
</PRE>

<P>
 where
  <ul>
   <li>START (14 characters) - not used by LigPlot<SUP>+</SUP>
    <ul>
     <li>Column 1: Alignment Position (dddd)
     <li>Column 2: No. of position selected for structural template (dddd)
     <li>Column 3: No. of proteins aligned at this position (dddd)
    </ul>
   <li>PROT 1,2,3... (11 characters per protein)
    <ul>
     <li>Column 4 (7,10 etc): Residue number in PDB file (ddddc) 4 digit number
         + 1 character insert code<BR>
     Importantly the insert code is always in the same position not within
     the 4 characters reserved for the pdb number (see below)
     <li>Column 5 (8,11 etc): Amino Acid Code (c)
     <li>Column 6 (9,12 etc): Secondary Structure Assignment (c)
    </ul>
   <li>END (18 characters) - not used by LigPlot<SUP>+</SUP>
    <ul>
     <li>Last Column-3: Consensus Secondary Structure Assignment (c)
     <li>Last Column-2: No. of alpha residues at this position (dddd
     <li>Last Column-1: No. of beta residues at this position (dddd)
     <li>Last Column: Structural Conservation Score (dd)
    </ul>
  </ul>
</ul>

<P>
<B>2. CAF format (.caf)</B>

<P>
This is one of the formats output by the CATHEDRAL structural
alignment program.

<PRE>
HH CATHEDRAL Alignment 2.02
CC Date:    Tue Feb 22 21:07:49 2011
CC Author:  cathedral
CC Protein1 Protein2 Len1 Len2 Score  Align %Ov  %Seq RMSD
RR   1byq    2wi7  213  209  93.05  208   97   99   1.38
RR   1byq    3d36  213  121  72.64  115   53   17   9.95
RR   2wi7    3d36  209  121  72.58  114   54   16   9.68

                 1         11        21        31        41        51        
pdb|3d36         --------------------VDIQATLAPFSVIGEREKFRQCLLNVMKNAIEAMPN----
pdb|3d36                              SSSSS         HHHHHHHHHHHHHHHHH        
pdb|1byq         PMEEEEVETFAFQAEIAQLMSLIINTFYS-------NK-EIFLRELISNSSDALDKIRYE
pdb|1byq               SSSSS  HHHHHHHHHHHH               HHHHHHHHHHHHHHHHHHHH
pdb|2wi7         -----EVETFAFQAEIAQLMSLIINTFYS-------NK-EIFLRELISNSSDALDKIRYE
pdb|2wi7                SSSS  HHHHHHHHHHHH               HHHHHHHHHHHHHHHHHHHH
                      ...............:::::::::       :: :::::::::*:::*:::....

---------+---------+---------+---------+---------+---------+---------+
1234567890123456789012345678901234567890123456789012345678901234567890
         1         2         3         4         5         6         7
</PRE>

<ul>
 <li>Header information - not used by LigPlot<SUP>+</SUP><BR>
  All-against-all structural similarity statistics.

 <li>The alignment given in chunks, with 2 lines per entry: the first
 gives the amino acid sequence of the protein and the second gives the
 secondary structure (which is not used by LigPlot<SUP>+</SUP>).
</ul>

<P>
<B>3. Multiple FASTA format (.fasta)</B>

<P>
The standard FASTA multiple alignment format:

<PRE>
>pdb|3d36
--------------------VDIQATLAPFSVIGEREKFRQCLLNVMKNAIEAMPN----
------------GGTLQVYVSI---DNGRVLIRIADTGVGMTKEQLERLGEPYFTTKGVK
G---------------TGLGMMVVYRIIES-MNGTIRIESEIH-----------------
----------------KGTTVSIYLPLAS-------------------------------
-------------
>pdb|1z5a
----------KEKFTSLSPAEFFKRNPELAGFPNPARALYQTVRELIENSLDATDVHGI-
------------LPNIKITIDLIDDARQIYKVNVVDNGIGIPPQEVPNAFGRVLYSSKYV
NRQTR---------GMYGLGVKAAVLYSQMHQDKPIEIETSPVNSKRIYTFKLKIDINKN
EPIIVERGSVENTRGFHGTSVAISI--PGDWPKAKSRIYEYIKRTYIITPYAEFIFKDPE
GNVTYYPRLTNKI
>pdb|1byq
PMEEEEVETFAFQAEIAQLMSLIINTFYS-------NK-EIFLRELISNSSDALDKIRYE
TLTDPSKLDSGKELHINLIPNKQD-----RTLTIVDTGIGMTKADLINNLGTIAKSGTKA
FMEALQAGADISMIGQFGVGFYSAYLVA-----EKVTVITKHNDD-EQYAWESSAG----
--GSFTVRTDTGEPMGRGTKVILHLKEDQTEYLEERRIKEIVKKHSQFI-GYPITLFVE-
-------------
</PRE>

The key thing here is the format of the protein name line which must
take one of the following forms:

<BLOCKQUOTE>
>pdb|1z5a<BR>
>pdb|1z5aA<BR>
>pdb|1z5aA01<BR>
</BLOCKQUOTE>

where "<B>A</B>" is the chain identifier and "<B>01</B>" is the domain
number.

</body>
